crl-test/service_accounts.tf

```diff
@@ -0,0 +1,24 @@
+# Service account for the CRL Integration tests to use.
+resource "google_service_account" "crl" {
+  project = var.google_project
```

Comment:
I would add depends_on bits for anything that depends on the project existing:
```
depends_on = [ google_project.project ]
```

You can also try and do the apply and see if TF is smart enough to figure this out, but I suspect not. One way to make it easier for it to figure it out would be to reference a property of the project rather than the variable:
```
project = google_project.project.name
```
If you do this you might not need the depends_on since this implies that.

Reply:
doh, thanks!